CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.14.0](https://github.com/aws-observability/aws-rum-web/compare/v1.12.0...v1.14.0) (2023-06-21)
+
+
+### Features
+
+* add ignore callback to resource plugin config ([3fd4d7b](https://github.com/aws-observability/aws-rum-web/commit/3fd4d7b184645e21e66f0ab199d3823c7a4435ee))
+* Create a session ID when cookies are disabled ([#314](https://github.com/aws-observability/aws-rum-web/issues/314)) ([6943587](https://github.com/aws-observability/aws-rum-web/commit/6943587259b8d7623742656199dc26b26b5bcd5d))
```

Comment:
It looks like some features are being duplicated from 1.13.0 and 1.14.0
Please double check what is included in this release that was not included 1.13.0 

Eg - referrer change went out in 1.13.0

Reply:
Removed the version bump